ckanext/data_qld/templates/admin/config.html

```diff
@@ -19,6 +19,19 @@
 
 {{ form.select('ckan.homepage_style', id='field-homepage-style', label=_('Homepage'), options=homepages, selected=data['ckan.homepage_style'], error=error) }}
 
+{% if h.profanity_checking_enabled() %}
```

Comment:
@ThrawnCA , do you know why we overide the admin form instead of using super() and adding our bits below it?

Reply:
only things missing is

```
 {{ form.select('ckan.main_css', id='field-ckan-main-css', label=_('Style'), options=styles, selected=data['ckan.main_css'], error=error) }}
 {{ form.textarea('ckan.site_custom_css', id='field-ckan-site-custom-css', label=_('Custom CSS'), value=data['ckan.site_custom_css'], error=error, placeholder=_('Customisable css inserted into the page header')) }}
 <div class="form-actions">
        <a href="{% url_for 'admin.reset_config' %}" class="btn btn-danger pull-left" data-module="confirm-action" data-module-content="{{ _('Are you sure you want to reset the config?') }}">{{ _('Reset') }}</a>
        <button type="submit" class="btn btn-primary" name="save">{{ _('Update Config') }}</button>
      </div>
```
and what we added was
```
<div class="form-actions">
        <a href="{% url_for 'admin.reset_config' %}" class="btn btn-danger pull-left" data-module="confirm-action" data-module-content="{{ _('Are you sure you want to reset the config?') }}">{{ _('Reset') }}</a>
        <button type="submit" class="btn btn-primary" name="save">{{ _('Update Config') }}</button>
      </div>
  </form>
<h3>Data Request Config</h3>

{{ form.markdown('ckanext.data_qld.datarequest_suggested_description', id='field-ckanext.data_qld.datarequest_suggested_description', label=_('Suggested Description'), placeholder=_('eg. Data Request suggested description'), value=data['ckanext.data_qld.datarequest_suggested_description'], error=errors['ckanext.data_qld.datarequest_suggested_description']) }}

<h3>Resource Config</h3>

{{ form.textarea('ckanext.data_qld.resource_formats', id='field-ckanext.data_qld.resource_formats', label=_('Resource Formats'), placeholder=_('eg. CSV, DOCX, HTML, PDF'), value=data['ckanext.data_qld.resource_formats'], error=errors['ckanext.data_qld.resource_formats']) }}

```

Reply:
We removed the styling options because TSS Cyber advised that they could be used for evil.